src/App.test.js

```diff
@@ -1,10 +1,32 @@
 import React from 'react'
-import { render, act } from '@testing-library/react'
+import { render, act, wait } from '@testing-library/react'
 import App from './App'
 
-test('renders sequence diagram', async () => {
+const mockJson = {
+  lifelines: [{ name: 'serviceA' }],
```

Comment:
maybe we'd be nice to extract this `"serviceA"` to a const?

Reply:
why? i did it and saw no gain of readability, if you're ok i'd rather keep the way it is

Reply:
because then you can use it when writing the test later (e.g. line 20). I just think that, in the test, when reading, for example, `SERVICE_NAME` makes more sense than just `serviceA`. But if you prefer this way, I'm ok.

Reply:
i got your point, i will change it